src/backend/pipeline/cont_combiner.c

```diff
@@ -995,6 +995,7 @@ ContinuousQueryCombinerMain(void)
 		}
 
 		CommitTransactionCommand();
+		pgstat_report_stat(true);
```

Comment:
Lets not force this. This function will automatically force once every `PGSTAT_STAT_INTERVAL`.